packages/common/src/services/resizer.service.ts

```diff
@@ -196,22 +223,25 @@ export class ResizerService {
       bottomPadding += parseInt(`${footerHeight}`, 10);
     }
 
-    let gridHeight = 0;
-    let gridOffsetTop = 0;
+    let gridHeight;
+    let gridWidth;
 
     // which DOM element are we using to calculate the available size for the grid?
-    if (autoResizeOptions.calculateAvailableSizeBy === 'container') {
+    if (autoResizeOptions.calculateAvailableSizeBy === 'container' || autoResizeOptions.useResizeObserver) {
       // uses the container's height to calculate grid height without any top offset
-      gridHeight = this._pageContainerElm.height() || 0;
+      gridHeight = this._pageContainerElm?.offsetHeight ?? 0;
+      gridWidth = this._pageContainerElm?.offsetWidth ?? 0;
     } else {
       // uses the browser's window height with its top offset to calculate grid height
       gridHeight = window.innerHeight || 0;
+      gridWidth = window.innerWidth || 0;
       const coordOffsetTop = this._gridDomElm.offset();
-      gridOffsetTop = (coordOffsetTop !== undefined) ? coordOffsetTop.top : 0;
+      const gridOffsetTop = (coordOffsetTop !== undefined) ? coordOffsetTop.top : 0;
+      gridHeight -= gridOffsetTop;
     }
 
-    const availableHeight = gridHeight - gridOffsetTop - bottomPadding;
-    const availableWidth = this._pageContainerElm.width() || window.innerWidth || 0;
+    const availableHeight = gridHeight - bottomPadding;
+    const availableWidth = gridWidth;
```

Comment:
the new `width` seems to be problematic as can be seen in the print screen

Reply:
My bad, sorry!

I think I got a little too enthusiastic with removing jQuery here without understanding exactly how jQuery determines width/height. This change was working fine in example-14 and in my application, but I didn't check all the other examples. 
I now looked into how jQuery determines width/height ... and it takes padding etc into account and has some more subtleties ... but in hindsight I don't really need this change for detecting resizes on elements, so I'll revert the code.